src/screens/ChannelScreen/index.js

```diff
@@ -50,7 +50,7 @@ const ChannelScreen = () => {
 
   return (
     <SafeAreaView>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to the `StatusBar` component is good as it sets the `barStyle` prop to `'light-content'`, which adjusts the appearance of the status bar. However, please ensure that this change is consistent across all screens for a uniform user experience.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```
This change will make the text and icons in the status bar light (white) which is suitable for dark backgrounds. If the background of the status bar on this screen is not dark, this change might make the status bar content hard to see.


<!-- This is an auto-generated comment by OSS CodeRabbit -->